docs-src/docs/reference/cli.md

```diff
@@ -556,44 +457,76 @@ Use the **create** subcommand to create a new load balancer.
 ####Optional arguments
 <div class="divTable">
   <div class="divRow">
-    <div class="divCellNoWrap">--port _hostPort:containerPort/protocol_</div>
-    <div class="divCell">The port configuration for the load balancer. _hostPort_ is the port on which the load balancer will listen for traffic; _containerPort_ is the port that traffic will be forwarded to. You can specify multiple ports using _--port xxx --port yyy_. If this option is not specified, Layer0 will use the following configuration: 80:80/tcp</div>
+    <div class="divCellNoWrap">
```

Comment:
All this HTML in markdown has always bugged me. Its like using a bike to get across the state when you own a car. Anyways, nothing that needs to be fixed in this PR - just something we may want to refactor if we have the time